mciem-member-projects.sh

```diff
@@ -1,9 +1,32 @@
 #!/bin/bash -ux
+export MCIEM_CUSTOM_ROLE_NAME=CloudKnoxIAMOrgAdmin-$(date +%s)
```

Comment:
ERROR: (gcloud.iam.roles.create) INVALID_ARGUMENT: The role_id "CloudKnoxIAMOrgAdmin-1653712691" is invalid. It doesn't match pattern "[a-zA-Z0-9_\.]{3,64}". The role_id must be 3 to 64 characters long and can be a mix of uppercase and lowercase English letters, digits, underscores and periods.